apollo-portal/src/main/resources/static/scripts/controller/role/ClusterNamespaceRoleController.js

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2024 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+role_module.controller('ClusterNamespaceRoleController',
+    ['$scope', '$location', '$window', '$translate', 'toastr', 'AppService', 'UserService', 'AppUtil', 'EnvService',
+      'PermissionService',
+      function ($scope, $location, $window, $translate, toastr, AppService, UserService, AppUtil, EnvService,
+          PermissionService) {
+
+        var params = AppUtil.parseParams($location.$$url);
+        $scope.pageContext = {
+          appId: params.appid,
+          env: params.env,
+          clusterName: params.clusterName
+        };
+
+        $scope.modifyRoleSubmitBtnDisabled = false;
+        $scope.ReleaseRoleSubmitBtnDisabled = false;
```

Comment:
Why is the variable in uppercase?
```suggestion
        $scope.releaseRoleSubmitBtnDisabled = false;
```